### DISCUSSION TITLE
Build Failure Type not found : T

### DISCUSSION THREAD

**Discussion by ghost** [FunkinCrew/Funkin#2815] [Compiling help] [answered]
I'm attempting to build the game using the instructions provided [here](https://github.com/FunkinCrew/Funkin/blob/main/docs/COMPILING.md), but I got an error that said:
`ERROR (unknown position)`
`    | Type not found : T`

I couldn't find much online about this, but I think it's caused by one of the T variables used for arrays. If anyone has any suggestions, please let me know. Thanks.

**Comment by NotHyper-474** [comment, marked as answer]
#2819 had the same error and reinstalling everything fixed it for them

**Reply by ghost** [reply]
I read through [#2819](https://github.com/FunkinCrew/Funkin/discussions/2819), and although reinstalling everything didn't work for me, `hmm reinstall -f flixel` and the commands provided [here](https://github.com/FunkinCrew/Funkin/discussions/2819#discussioncomment-9782786) worked for me. Thank you so much.
